tls/s2n_handshake.c

```diff
@@ -100,3 +101,61 @@ int s2n_handshake_get_hash_state(struct s2n_connection *conn, s2n_hash_algorithm
     }
     return 0;
 }
+
+int s2n_handshake_require_all_hashes(struct s2n_connection *conn)
```

Comment:
`s2n_handshake_require_all_hashes` takes an `s2n_connection` parameter, whereas `s2n_handshake_is_hash_required` takes an `s2n_handshake` parameter. Are you opposed to updating this to take an `s2n_handshake` parameter to match behavior?

Reply:
Not opposed at all! Good suggestion. Updated.